scattr/workflow/rules/qc.smk

```diff
@@ -71,7 +71,7 @@ rule registration_qc:
         fixed_nii=lambda wildcards: expand(
             inputs["T1w"].path,
             zip,
-            **filter_list(inputs["T1w"].input_zip_lists, wildcards)
+            **filter_list(inputs["T1w"].zip_lists, wildcards)
```

Comment:
I think we could make this `fixed_nii=lambda wildcards: inputs["T1w"].filter(**wildcards).expand()`